llvm/test/Analysis/CostModel/RISCV/abs.ll

```diff
@@ -1,6 +1,6 @@
 ; NOTE: Assertions have been autogenerated by utils/update_analyze_test_checks.py
 ; RUN: opt < %s -passes="print<cost-model>" 2>&1 -disable-output -S -mtriple=riscv64 -mattr=+v | FileCheck %s
-; Check that we don't crash querying costs when vectors are not enabled.
```

Comment:
Why remove this comment?

Reply:
Sorry, I accidentally deleted it, I have modified it back